data/gui/widget/text_box_default.cfg

```diff
@@ -100,7 +103,7 @@
 		font_family = {FONT_FAMILY}
 
 		color = "(
-			if(text = '' and hint_text != '', hint_color, reg_color) where
+			if((text = '' and hint_text != '') or editable = 0, hint_color, reg_color) where
```

Comment:
I think all the changes to text_box_default.cfg go with the editability feature.

Reply:
Yes, I just noticed that a couple of minutes ago. Will be removed.

Reply:
Removed.